ocaml/xenopsd/scripts/swtpm-wrapper

```diff
@@ -132,21 +176,32 @@ def main(argv):
             os.chown(tpm_dir, uid, uid)
             if os.path.exists(os.path.join(tpm_dir, ".lock")):
                 os.chown(os.path.join(tpm_dir, ".lock"), uid, uid)
-            os.chown(os.path.join(tpm_dir, STATE_FILE), uid, uid)
+            os.chown(tpm_file, uid, uid)
 
         except OSError as error:
             print(error)
             return
 
         tpm_path = '/'
+        tpm_uri = tpm_state
+
+        if (tpm_state_prefix == "file"):
```

Comment:
Should this be in the `if depriv` section?

Reply:
It all about restricting priveladge, so yes?